From: Dima Kogan <dima@secretsauce.net>
Date: Sun, 27 Apr 2014 00:41:13 -0700
Subject: x11 terminal clipboard paste: can now paste the coordinates more
 than once

For some reason, gnuplot was clearing out the clipboard with every paste. So
applications could only paste coordinates from gnuplot once.
---
 src/gplt_x11.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/gplt_x11.c b/src/gplt_x11.c
index a83cb87..84fb7df 100644
--- a/src/gplt_x11.c
+++ b/src/gplt_x11.c
@@ -6484,7 +6484,6 @@ handle_selection_event(XEvent *event)
 		XChangeProperty(dpy, reply.xselection.requestor,
 				reply.xselection.property, reply.xselection.target,
 				8, PropModeReplace, (unsigned char *) selection, strlen(selection));
-		*selection = '\0';
 #endif
 	    } else {
 		FPRINTF((stderr, "selection request target: %d\n", reply.xselection.target));
